src/platform/monitoring/DowntimeNotification/containers/DowntimeNotification.jsx

```diff
@@ -99,6 +99,7 @@ class DowntimeNotification extends React.Component {
     if (this.props.render) {
       return this.props.render(
         {
+          appTitle: this.props.appTitle,
```

Comment:
Looks like you may have to clean up all this lint warnings

Reply:
I think a refactor of `src/platform/monitoring/DowntimeNotification/` is probably needed, given there is more going on that could use some work. For example, [removing dependency on momentjs](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75609). I'd also like to see @jzucadi's ["refactor to a functional component" PR](https://github.com/department-of-veterans-affairs/vets-website/pull/27215) get picked back up